nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'Method not implemented'
+  end
```

Comment:
[OPTIONAL] For better practice, It would be better to mention the class and the method in the error message to be easier for you to debug your code and to know the type of the error.
```ruby
class Nameable
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
end
```

Reply:
✅ RESOLVED: Update `correct_name` method in `Nameable` class to raise error that mentions the class/method.